pmlc/conversion/linalg_to_pxa/test/conv.mlir

```diff
@@ -0,0 +1,20 @@
+// RUN: pmlc-opt -convert-linalg-to-pxa %s | FileCheck %s
+
+module  {
+  func @test_conv(%arg0: memref<128x128x3xf32, offset: ?, strides: [?, ?, 1]>, %arg1: memref<3x3x3xf32, offset: ?, strides: [?, ?, 1]>, %arg2: memref<?x?x?xf32, offset: ?, strides: [?, ?, 1]>) {
+    linalg.conv(%arg0, %arg1, %arg2) {strides = [2]}: memref<128x128x3xf32, offset: ?, strides: [?, ?, 1]>, memref<3x3x3xf32, offset: ?, strides: [?, ?, 1]>, memref<?x?x?xf32, offset: ?, strides: [?, ?, 1]>
+    return
+  }
+}
+
+// CHECK: #map = affine_map<(d0, d1, d2)[s0, s1, s2] -> (d0 * s1 + s0 + d1 * s2 + d2)>
+// CHECK-LABEL: func @test_conv
+// CHECK-SAME: (%[[arg0:.*]]: memref<128x128x3xf32, #map>, %[[arg1:.*]]: memref<3x3x3xf32, #map>, %[[arg2:.*]]: memref<?x?x?xf32, #map>)
+// CHECK: affine.parallel (%[[arg3:.*]], %[[arg4:.*]], %[[arg5:.*]], %[[arg6:.*]], %[[arg7:.*]]) = (0, 0, 0, 0, 0) to (3, -1, 3, 128, 128) reduce ("addf")
```

Comment:
Is it legal for the bounds to go from `0` to `-1`? That seems odd to me.

Reply:
I think that's because of ? which was removed.